avro-fastserde/src/test/java/com/linkedin/avro/fastserde/FastSerdeTestsSupport.java

```diff
@@ -186,4 +190,90 @@ public static File getCodeGenDirectory() throws IOException {
     }
     return dir;
   }
+
+  public static void setField(SpecificRecord sr, String fieldName, Object value) {
+    Class srClass = sr.getClass();
+    try {
+      Field field = srClass.getField(fieldName);
```

Comment:
what happens if the field is private?

Reply:
It throws the NoSuchFieldException and proceeds to trying to find the setter function instead.